ephios/core/signup/methods.py

```diff
@@ -458,6 +469,10 @@ class BaseSignupMethodConfigurationForm(forms.Form):
         initial=True,
     )
 
+    def __init__(self, *args, **kwargs):
+        self.event = kwargs.pop("event", None)
```

Comment:
\*sigh*
I don't like that we need to introduce the event parameter here but I guess it makes sense. In the usage down below you just expect `self.event` to not be None, so this should really be:
```suggestion
    def __init__(self, *args, **kwargs):
        self.event = kwargs.pop("event")
```